src/cr/cube/cubepart.py

```diff
@@ -669,6 +703,18 @@ def unweighted_bases(self):
         """
         return tuple(np.broadcast_to(self.table_base, self._shape))
 
+    @lazyproperty
+    def var_scale_mean(self):
+        if np.all(np.isnan(self._numeric_values)):
+            return None
+
+        numeric_values_masked, is_a_number_mask = self._numeric_values_masked
+        counts = self._counts_as_array[is_a_number_mask]
```

Comment:
The counts are indexed here, while the numeric values are indexed in a different place. But still using the same mask. I'd prefer if the indexing was done in the same place, so it's more explicit and concise. I.e. maybe create a single property for `_numeric_mask`, and then use that from within this method, to get both the numeric values and counts:
```
num_vals = self._numeric_values[self._numeric_mask]
counts = self._counts_as_array[self._numeric_mask]
```
The way it is now, you kinda have to swap back and forth between two different functions, to figure out what's happening.

Reply:
u right ... fixed in here 32cab83a2e48c62179fa4b764eebf72629a22750